service_thing_aggregate/src/main/resources/evolutions/default/1.sql

```diff
@@ -23,3 +23,13 @@ CREATE TABLE MUSARK_STORAGE.STORAGE_NODE (
  museum_id INTEGER NOT NULL,
  PRIMARY KEY (storage_node_id)
 );
+
+CREATE TABLE MUSIT_MAPPING.MUSITTHING (
```

Comment:
We have an agreement between cultural and natural history to use the term 'Object' (in some cases 'museum object') as a common term. 'Thing' was intended as a common term for objects, documents, photographs, and so forth.